### PR TITLE
feat(about): add Addison and Lorenzo as volunteers

### PR DESCRIPTION
**Pull request by dergigi**
To be reviewed:

- [x] [/about/addison](https://os-website-git-more-volunteers-open-sats.vercel.app/about/addison)
- [ ] ~~[/about/constant](https://os-website-git-more-volunteers-open-sats.vercel.app/about/constant)~~
- [x] [/about/lorenzo](https://os-website-git-more-volunteers-open-sats.vercel.app/about/lorenzo)

TODO:

- [ ] ~~Add Constant's bio~~

We'll have to update Constant's bio once he's back from holiday. I'll merge the other two.

### CODE REVIEW

File: data/authors/lorenzo.mdx
Comment (lollodev) [inline data/authors/lorenzo.mdx:1]: Approved, thank you!

File: data/authors/constant.mdx
Comment (lollodev) [inline data/authors/constant.mdx:1]: Sorry not my bio, but is this a placeholder or intentional? I think this is default text showing in the bio

**Review by lollodev — APPROVED** [review]
Reviewed Lorenz bio + inserted comment for c0nst4nt, please check it

File: data/authors/constant.mdx
Comment (dergigi) [reply data/authors/constant.mdx:1]: Yes, that's intentional. Will be updated soon 👍

**Review by bitcoinprecept — APPROVED** [review]
LGTM

File: data/authors/constant.mdx
Comment (bitcoinprecept) [reply data/authors/constant.mdx:1]: LGTM

**Review by bitcoinprecept — APPROVED** [review]
Forgive me if this is duplicative - getting oriented to some new aspects of Github.

Looks good.

**Pull request merged**